packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -75,20 +75,19 @@ export function addRoutesToApp(app: Express) {
    */
   app.post('/v1/login', handleWith(LoginRoute));
   app.post('/v1/logout', handleWith(LogoutRoute));
-  app.post(
-    '/v1/confirmedWeeklyReport/:organisationUnitCode',
-    handleWith(ConfirmCountryWeeklyReportRoute),
-  );
+  app.post('/v1/confirmedWeeklyReport/:countryCode', handleWith(ConfirmCountryWeeklyReportRoute));
 
   /**
    * PUT routes
    */
-  app.put('/v1/weeklyReport/:organisationUnitCode', handleWith(SaveCountryWeeklyReportRoute));
+  app.put('/v1/weeklyReport/:countryCode', handleWith(SaveWeeklyReportRoute));
+  app.put('/v1/weeklyReport/:countryCode/:siteCode', handleWith(SaveWeeklyReportRoute));
```

Comment:
Optional, and not 100% sure about the syntax so you may need to test this, but I think you can indicate an optional param like this: 
`app.put('/v1/weeklyReport/:countryCode/:siteCode?', handleWith(SaveWeeklyReportRoute));`

Which should match both: `/v1/weeklyReport/TO` and `/v1/weeklyReport/TO/TO_Nukuhc` and assign params correctly

Reply:
That would be great! Thanks for the suggestion, I'll try this out